src/reply_maker.js

```diff
@@ -17,22 +18,31 @@ function formatReply(comment, conversions) {
     transform = (x) => x.toUpperCase();
   }
 
+  const items = [
+    {"value" : "metric units " + species },
+    {"type" : "link", "value" : "feedback", "href" : "https://redd.it/73edn2" },
+    {"type" : "link", "value" : "source", "href" : "https://github.com/cannawen/metric_units_reddit_bot" },
+    {"type" : "link", "value" : "hacktoberfest", "href" : "https://redd.it/73ef7e" },
+    {"type" : "link", "value" : "block", "href" : formatRedditComposeLink(environment['reddit-username'], 'block', stopMessage) },
+    {"type" : "link", "value" : "refresh conversion", "href" : formatRedditComposeLink(environment['reddit-username'], 'refresh ' + commentId, "Please click 'send' below and I will update my comment to convert any new or updated values in your comment.") },
+    {"value" : version }
+  ];
+
+  const footer = items.map(item => {
+    item.value = transform(item.value);
+    item.value = '^' + item.value.replace(/ /g, ' ^');
+    if (item.type == 'link') item.value = `[${item.value}](${item.href})`;
+    return item.value;
+  }).join(' ^| ');
```

Comment:
This refactor looks awesome!!!!! Much easier to read and understand what is going on